src/Moryx.DependentTestModule/ModuleController/ModuleController.cs

```diff
@@ -23,8 +24,8 @@ public class ModuleController : ServerModuleFacadeControllerBase<ModuleConfig>,
         [RequiredModuleApi(IsStartDependency = true, IsOptional = false)]
         public ITestModule TestModule { get; set; }
 
-        public ModuleController(IModuleContainerFactory containerFactory, IConfigManager configManager, IServerLoggerManagement loggerManagement) 
-            : base(containerFactory, configManager, loggerManagement)
+        public ModuleController(IModuleContainerFactory containerFactory, IConfigManager configManager, ILoggerFactory loggerFactory) 
```

Comment:
Could we create something like a MigrationGuide document where we gather the necessary changes for the next major?

Reply:
Good point